F/FXdiv/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FXdiv"
+version = v"0.0.20181117"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Maratyszcza/FXdiv.git", "b742d1143724d646cd0f914646f1240eacf5bd73"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/FXdiv
+mkdir $includedir
+cp -a include/fxdiv.h $includedir
```

Comment:
```suggestion
install -Dvm 644 include/fxdiv.h "${includedir}/fxdiv.h"
```

Reply:
Never call `cp` in this way ([second time today](https://github.com/JuliaPackaging/Yggdrasil/pull/4555#r820271910) I think?).  The problem is that if the second argument doesn't exist, you'll create a file with that name, instead of inside that directory.  Yes, here you created the directory at the line above, but it's easy to forget to create it and this happens continuously here, so a good rule of thumb is to just don't do that.  Either do something like
```sh
cp include/fxdiv.h $includedir/
cp include/fxdiv.h $includedir/.
cp include/fxdiv.h $includedir/fxdiv.h
```
or just use `install -D` which does the right thing and also automatically create the parent directories as needed.

Reply:
Right - thanks for the explanation - after the first it wasn't completely clear why `cp -a` was a bad thing. But yes - I've seen my share of mishaps like that as well (which is why I usually remember to get the directories in place :-)

But nice to know that `install -D` can help out.